pkgs/build-support/cc-wrapper/ld-wrapper.sh

```diff
@@ -71,8 +71,18 @@ declare -a libDirs
 declare -A libs
 relocatable=
```

Comment:
It seems that `link32` may be left undefined. I'd suggest declaring it here and moving these declarations under the following comment.

Reply:
I'm surprised I didn't hit that with `set -u`.